moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1984,7 +1991,7 @@ double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* gro
   {
     std::vector<RobotStatePtr> waypoint_traj;
     double wp_percentage_solved = computeCartesianPath(group, waypoint_traj, link, waypoints[i], global_reference_frame,
-                                                       max_step, jump_threshold, validCallback, options);
+                                                       max_step, 0.0, validCallback, options);
```

Comment:
move this 0.0 into a named ``static const`` with a comment explaining that you don't want to perform joint space jump testing until the entire trajectory is generated